narwhal/storage/src/node_store.rs

```diff
@@ -59,11 +59,12 @@ impl NodeStorage {
         store_path: Path,
         certificate_store_cache_metrics: Option<CertificateStoreCacheMetrics>,
     ) -> Self {
+        let db_options = default_db_options().optimize_for_write_throughput(8);
```

Comment:
Do we care optimising for read (scans- `optimize_for_read`) as well ? I could see this benefit the read from tables like the `CERTIFICATE_DIGEST_BY_ORIGIN_CF`

Reply:
That can be a separate change.